code/src/components/HappyHeart.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react' 
+import './happyHeart.css'
+
```

Comment:
Making the heart-button to a separate component cleans up the code in the HappyThought-component. Smart of you!